src/main/java/com/iota/iri/BundleValidator.java

```diff
@@ -8,8 +8,50 @@
 
 import java.util.*;
 
+/**
+ * Validates bundles.
+ * <p>
+ * Bundles are a list of transactions that represent an atomic transfer, meaning that either all
```

Comment:
```suggestion
 * Bundles are lists of transactions that represent an atomic transfer, meaning that either all
```

Reply:
Either all plural or 'A bundle is a list of transactions'. The common pattern in docs is to use plural though.